### ISSUE TITLE
[Enhancement] create_emr_cluster failed  version  > 6.8.0

### ISSUE THREAD

**Issue by LLejoly** [aws/aws-sdk-pandas#2162]
When you try to instantiate a new cluster with

```
 wr.emr.create_cluster(subnet_id=subnet_id,
                                       cluster_name=cluster_name,
                                       emr_release='emr-6.9.0')
```

It will failed with the following error:

`botocore.exceptions.ClientError: An error occurred (ValidationException) when calling the RunJobFlow operation: Classification 'spark-log4j' is not valid for parent classification 'null'.`

It is because the configurations loaded by default is not valid anymore for version > 6.7.x

https://github.com/aws/aws-sdk-pandas/blob/a279ce9e538905bd22c66031d657c762218e5b38/awswrangler/emr.py#L157-166

A dynamic configurations loading should be implemented in order to avoid the error depicted above.

A potential quick solution could be:

```
  # Configurations
  args["Configurations"] = (
      [
          {
              "Classification": "spark-log4j",
              "Properties": {"log4j.rootCategory": f"{pars['spark_log_level']}, console"},
          }
      ]
      if not pars["configurations"] and pars["emr_release"] in (supported_versions)
      [
          {
              "Classification": "spark-log4j2",
              "Properties": {"log4j.rootCategory": f"{pars['spark_log_level']}, console"},
          }
      ]
      elif  not pars["configurations"] and pars["emr_release"] in (supported_versions)
      else pars["configurations"]
  )
```

**Comment by LLejoly** [comment]
I will create a pull request when I have time to work on it

**Comment by jaidisido** [comment]
Hi @LLejoly, making sure this is not a duplicate of #1938 which should have been fixed in #2065. Could it be that you are not using the most recent version of the library?
